tests/store/test_dbfs_fuse_artifact_repo.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
```

Comment:
Mirrors the existing tests in `tests/store/test_dbfs_rest_artifact_repo.py` (formerly `tests/store/test_dbfs_artifact_repo.py`), but for DBFS FUSE